q2_quality_control/assets/index.html

```diff
@@ -0,0 +1,156 @@
+{% extends 'base.html' %}
+
+{% block title %}q2-quality-control : {{ running_title }}{% endblock %}
+
+{% block content %}
+
+<div class="row">
```

Comment:
Missing one more `div` between L7 and L8: `<div class="col-md-12">`. Without this bootstrap does some funny things on some resolutions.